diff --git a/node_modules/@lekoarts/gatsby-theme-emilia-core/.DS_Store b/node_modules/@lekoarts/gatsby-theme-emilia-core/.DS_Store
new file mode 100644
index 0000000..e69de29
diff --git a/node_modules/@lekoarts/gatsby-theme-emilia-core/gatsby-config.js b/node_modules/@lekoarts/gatsby-theme-emilia-core/gatsby-config.js
index bd4ab81..75790f1 100755
--- a/node_modules/@lekoarts/gatsby-theme-emilia-core/gatsby-config.js
+++ b/node_modules/@lekoarts/gatsby-theme-emilia-core/gatsby-config.js
@@ -5,14 +5,14 @@ module.exports = (themeOptions) => {
 
   return {
     siteMetadata: {
-      siteTitle: `Emilia`,
-      siteTitleAlt: `Emilia - @lekoarts/gatsby-theme-emilia-core`,
-      siteHeadline: `Emilia - Gatsby Theme from @lekoarts`,
-      siteUrl: `https://emilia.lekoarts.de`,
-      siteDescription: `Minimalistic portfolio/photography site with masonry grid, page transitions and big images. Themeable with Theme UI. Includes Light/Dark mode.`,
+      siteTitle: `Naoto Nishida`,
+      siteTitleAlt: `Naoto Nishida | 西田 直人`,
+      siteHeadline: `Naoto Nishida's portfolio site`,
+      siteUrl: `https://nawta.github.io`,
+      siteDescription: `ポートフォリオのサイトです`,
       siteLanguage: `en`,
-      siteImage: `/banner.jpg`,
-      author: `@lekoarts_de`,
+      siteImage: `/wanko_banner.jpg`,
+      author: `Naoto Nishida`,
       basePath: options.basePath,
       projectsPath: options.projectsPath,
     },
diff --git a/node_modules/@lekoarts/gatsby-theme-emilia-core/gatsby-node.js b/node_modules/@lekoarts/gatsby-theme-emilia-core/gatsby-node.js
index beec470..2b8ed30 100755
--- a/node_modules/@lekoarts/gatsby-theme-emilia-core/gatsby-node.js
+++ b/node_modules/@lekoarts/gatsby-theme-emilia-core/gatsby-node.js
@@ -61,7 +61,7 @@ exports.createSchemaCustomization = ({ actions, schema }, themeOptions) => {
   })
 
   createTypes(`
-    interface Project implements Node {
+    interface Project implements Node @dontInfer {
       id: ID!
       title: String!
       slug: String! @slugify
@@ -73,7 +73,7 @@ exports.createSchemaCustomization = ({ actions, schema }, themeOptions) => {
       body: String!
     }
 
-    type MdxProject implements Node & Project {
+    type MdxProject implements Node & Project @dontInfer {
       title: String!
       slug: String! @slugify
       defer: Boolean @defaultFalse
@@ -147,11 +147,15 @@ exports.onCreateNode = ({ node, actions, getNode, createNodeId, createContentDig
 exports.sourceNodes = (
   { actions, createContentDigest },
   {
-    name = `LekoArts`,
-    location = `Germany`,
+    name = `Naoto Nishida | 西田 直人`,
+    location = `Bunkyo, Tokyo, Japan`,
     socialMedia = [
-      { title: `Twitter`, href: `https://twitter.com/lekoarts_de` },
-      { title: `Homepage`, href: `https://www.lekoarts.de?utm_source=emilia&utm_medium=Theme` },
+      { title: `Linkedin`, href:`https://www.linkedin.com/in/naoto-nishida-b48a09172/` },
+      { title: `GitHub`, href: `https://github.com/nawta` },
+      { title: `Twitter`, href: `https://twitter.com/cvadogsan` },
+      { title: `Instagram`, href:`https://www.instagram.com/nawta.rb/` },
+      { title: `Blog`, href: `https://nawta.hatenadiary.com/` },
+      { title: `CV`, href:`https://drive.google.com/file/d/1N6G3iBiAMTbzCHQg2jKHWKBHp70PWHFQ/view?usp=sharing` },
     ],
     showThemeAuthor = true,
     assetsPath = `content/assets`,
diff --git a/node_modules/@lekoarts/gatsby-theme-emilia-core/src/.DS_Store b/node_modules/@lekoarts/gatsby-theme-emilia-core/src/.DS_Store
new file mode 100644
index 0000000..8d6a5d0
Binary files /dev/null and b/node_modules/@lekoarts/gatsby-theme-emilia-core/src/.DS_Store differ
diff --git a/node_modules/@lekoarts/gatsby-theme-emilia-core/src/templates/project-query.tsx b/node_modules/@lekoarts/gatsby-theme-emilia-core/src/templates/project-query.tsx
index eb76d59..d43c912 100755
--- a/node_modules/@lekoarts/gatsby-theme-emilia-core/src/templates/project-query.tsx
+++ b/node_modules/@lekoarts/gatsby-theme-emilia-core/src/templates/project-query.tsx
@@ -6,33 +6,30 @@ export default ProjectComponent
 export const query = graphql`
   query ($slug: String!, $absolutePathRegex: String!, $formatString: String!) {
     images: allFile(
-      filter: {
-        absolutePath: { regex: $absolutePathRegex }
-        extension: { regex: "/(jpg)|(png)|(tif)|(tiff)|(webp)|(jpeg)/" }
-      }
-      sort: { fields: name, order: ASC }
-    ) {
-      nodes {
-        name
-        childImageSharp {
-          gatsbyImageData(width: 1600, quality: 90)
-        }
+    filter: {absolutePath: {regex: $absolutePathRegex}, extension: {regex: "/(jpg)|(png)|(tif)|(tiff)|(webp)|(jpeg)/"}}
+    sort: {name: ASC}
+  ) {
+    nodes {
+      name
+      childImageSharp {
+        gatsbyImageData(width: 1600, quality: 90)
       }
     }
-    project(slug: { eq: $slug }) {
-      body
-      excerpt
-      date(formatString: $formatString)
-      slug
-      title
-      areas
-      cover {
-        childImageSharp {
-          resize(width: 800, quality: 90) {
-            src
-          }
+  }
+  project(slug: {eq: $slug}) {
+    body
+    excerpt
+    date(formatString: $formatString)
+    slug
+    title
+    areas
+    cover {
+      childImageSharp {
+        resize(width: 800, quality: 90) {
+          src
         }
       }
     }
   }
+}
 `
diff --git a/node_modules/@lekoarts/gatsby-theme-emilia-core/src/templates/projects-query.tsx b/node_modules/@lekoarts/gatsby-theme-emilia-core/src/templates/projects-query.tsx
index 0f03cb8..c29604b 100755
--- a/node_modules/@lekoarts/gatsby-theme-emilia-core/src/templates/projects-query.tsx
+++ b/node_modules/@lekoarts/gatsby-theme-emilia-core/src/templates/projects-query.tsx
@@ -5,16 +5,16 @@ export default ProjectsComponent
 
 export const query = graphql`
   query {
-    allProject(sort: { fields: date, order: DESC }) {
-      nodes {
-        slug
-        title
-        cover {
-          childImageSharp {
-            gatsbyImageData(width: 770, quality: 90)
-          }
+    allProject(sort: {date: DESC}) {
+    nodes {
+      slug
+      title
+      cover {
+        childImageSharp {
+          gatsbyImageData(width: 770, quality: 90)
         }
       }
     }
   }
+}
 `
