diff --git a/node_modules/@lekoarts/gatsby-theme-emilia/.DS_Store b/node_modules/@lekoarts/gatsby-theme-emilia/.DS_Store
new file mode 100644
index 0000000..9a874b5
Binary files /dev/null and b/node_modules/@lekoarts/gatsby-theme-emilia/.DS_Store differ
diff --git a/node_modules/@lekoarts/gatsby-theme-emilia/src/.DS_Store b/node_modules/@lekoarts/gatsby-theme-emilia/src/.DS_Store
new file mode 100644
index 0000000..6dd5571
Binary files /dev/null and b/node_modules/@lekoarts/gatsby-theme-emilia/src/.DS_Store differ
diff --git a/node_modules/@lekoarts/gatsby-theme-emilia/src/@lekoarts/.DS_Store b/node_modules/@lekoarts/gatsby-theme-emilia/src/@lekoarts/.DS_Store
new file mode 100644
index 0000000..b1f02aa
Binary files /dev/null and b/node_modules/@lekoarts/gatsby-theme-emilia/src/@lekoarts/.DS_Store differ
diff --git a/node_modules/@lekoarts/gatsby-theme-emilia/src/@lekoarts/gatsby-theme-emilia-core/.DS_Store b/node_modules/@lekoarts/gatsby-theme-emilia/src/@lekoarts/gatsby-theme-emilia-core/.DS_Store
new file mode 100644
index 0000000..7ceb314
Binary files /dev/null and b/node_modules/@lekoarts/gatsby-theme-emilia/src/@lekoarts/gatsby-theme-emilia-core/.DS_Store differ
diff --git a/node_modules/@lekoarts/gatsby-theme-emilia/src/components/footer.tsx b/node_modules/@lekoarts/gatsby-theme-emilia/src/components/footer.tsx
index 56d6123..3bad814 100755
--- a/node_modules/@lekoarts/gatsby-theme-emilia/src/components/footer.tsx
+++ b/node_modules/@lekoarts/gatsby-theme-emilia/src/components/footer.tsx
@@ -48,46 +48,6 @@ const Footer = () => {
           </Flex>
         </div>
       </Container>
-      {showThemeAuthor && (
-        <Container
-          sx={{
-            display: `flex`,
-            justifyContent: `center`,
-            alignItems: `center`,
-            color: `text`,
-            fontWeight: `semibold`,
-            a: { color: `text` },
-            mt: 4,
-          }}
-        >
-          {isDark ? (
-            <img
-              width="30"
-              height="30"
-              src="https://img.lekoarts.de/gatsby/logo_v2-light_w30.png"
-              alt="LekoArts Logo"
-            />
-          ) : (
-            <img width="30" height="30" src="https://img.lekoarts.de/gatsby/logo_v2_w30.png" alt="LekoArts Logo" />
-          )}
-          {` `}
-          <Link
-            aria-label="Link to the theme's GitHub repository"
-            sx={{ ml: 2 }}
-            href="https://github.com/LekoArts/gatsby-themes/tree/main/themes/gatsby-theme-emilia"
-          >
-            Theme
-          </Link>
-          <div sx={{ mx: 1 }}>by</div>
-          {` `}
-          <Link
-            aria-label="Link to the theme author's website"
-            href="https://www.lekoarts.de?utm_source=emilia&utm_medium=Theme"
-          >
-            LekoArts
-          </Link>
-        </Container>
-      )}
     </Box>
   )
 }
diff --git a/node_modules/@lekoarts/gatsby-theme-emilia/src/components/projects.tsx b/node_modules/@lekoarts/gatsby-theme-emilia/src/components/projects.tsx
index 2b583cf..8d4e022 100755
--- a/node_modules/@lekoarts/gatsby-theme-emilia/src/components/projects.tsx
+++ b/node_modules/@lekoarts/gatsby-theme-emilia/src/components/projects.tsx
@@ -75,7 +75,7 @@ const Projects = ({ projects }: Props) => {
             sx={{
               mt: `-8rem`,
               display: `grid`,
-              gridTemplateColumns: [`1fr`, `repeat(auto-fill, minmax(350px, 1fr))`],
+              gridTemplateColumns: [`1fr`, `repeat(auto-fill, minmax(200px, 1fr))`],
               gridGap: 4,
               alignItems: `flex-start`,
             }}
diff --git a/node_modules/@lekoarts/gatsby-theme-emilia/src/components/seo.tsx b/node_modules/@lekoarts/gatsby-theme-emilia/src/components/seo.tsx
index d6f186c..93c1f37 100755
--- a/node_modules/@lekoarts/gatsby-theme-emilia/src/components/seo.tsx
+++ b/node_modules/@lekoarts/gatsby-theme-emilia/src/components/seo.tsx
@@ -49,9 +49,9 @@ const Seo = ({ title = ``, description = ``, pathname = ``, image = ``, children
       <meta name="twitter:image:alt" content={seo.description} />
       <meta name="twitter:creator" content={author} />
       <meta name="gatsby-theme" content="@lekoarts/gatsby-theme-emilia" />
-      <link rel="icon" type="image/png" sizes="32x32" href={withPrefix(`/favicon-32x32.png`)} />
-      <link rel="icon" type="image/png" sizes="16x16" href={withPrefix(`/favicon-16x16.png`)} />
-      <link rel="apple-touch-icon" sizes="180x180" href={withPrefix(`/apple-touch-icon.png`)} />
+      <link rel="icon" type="image/png" sizes="32x32" href={withPrefix(`/inu-favicon-32x32.png`)} />
+      <link rel="icon" type="image/png" sizes="16x16" href={withPrefix(`/inu-favicon-16x16.png`)} />
+      <link rel="apple-touch-icon" sizes="180x180" href={withPrefix(`/penguin-apple-touch-icon.png`)} />
       {children}
     </>
   )
diff --git a/node_modules/@lekoarts/gatsby-theme-emilia/src/texts/about-me-en.mdx b/node_modules/@lekoarts/gatsby-theme-emilia/src/texts/about-me-en.mdx
new file mode 100644
index 0000000..a89755f
--- /dev/null
+++ b/node_modules/@lekoarts/gatsby-theme-emilia/src/texts/about-me-en.mdx
@@ -0,0 +1,3 @@
+## About Me
+
+Boggarts lavender robes, Hermione Granger Fantastic Beasts and Where to Find Them. Bee in your bonnet Hand of Glory elder wand, spectacles House Cup Bertie Bott’s Every Flavor Beans Impedimenta. Stunning spells tap-dancing spider Slytherin’s Heir mewing kittens Remus Lupin. Palominos scarlet train black robes, Metamorphimagus Niffler dead easy second bedroom. Padma and Parvati Sorting Hat Minister of Magic blue turban remember my last.
diff --git a/node_modules/@lekoarts/gatsby-theme-emilia/src/texts/about-me-ja.mdx b/node_modules/@lekoarts/gatsby-theme-emilia/src/texts/about-me-ja.mdx
new file mode 100644
index 0000000..a89755f
--- /dev/null
+++ b/node_modules/@lekoarts/gatsby-theme-emilia/src/texts/about-me-ja.mdx
@@ -0,0 +1,3 @@
+## About Me
+
+Boggarts lavender robes, Hermione Granger Fantastic Beasts and Where to Find Them. Bee in your bonnet Hand of Glory elder wand, spectacles House Cup Bertie Bott’s Every Flavor Beans Impedimenta. Stunning spells tap-dancing spider Slytherin’s Heir mewing kittens Remus Lupin. Palominos scarlet train black robes, Metamorphimagus Niffler dead easy second bedroom. Padma and Parvati Sorting Hat Minister of Magic blue turban remember my last.
